### PR TITLE
added led offset param to torso light config

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

**Comment by floweisshardt** [comment]
LGTM

**Comment by NadiaHG** [comment]
LGTM

File: cob_hardware_config/cob4-2/config/light_torso.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ devicestring: /dev/ttyLED
 devicedriver: stageprofi
 baudrate: 38400
 num_leds: 58
+led_offset: 5
```
Comment (fmessmer) [inline cob_hardware_config/cob4-2/config/light_torso.yaml:5]: Is there a default value `0`?
Do the light yamls for the other robots need this new parameter, too?

Comment (benmaidel) [inline cob_hardware_config/cob4-2/config/light_torso.yaml:5]: nope, they don't. Default is always 0.

**Pull request merged**